tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -56,6 +56,10 @@ def loganalyzer(duthosts, request):
         analyzers[duthost.hostname] = LogAnalyzer(ansible_host=duthost, marker_prefix=request.node.name)
     markers = parallel_run(analyzer_add_marker, [analyzers], {}, duthosts, timeout=120)
 
+    # Force load regular expressions from common files
+    for host in analyzers:
+        analyzers[host].load_common_config()
```

Comment:
It is better to run "load_common_config" in lines 55-56 while creating analyzer?

And line 33-35 should be removed.

Reply:
Hi, @wangxin If add "load_common_config" into line 55-56 - self.match_regex will be empty after line 57.
"load_common_config" - should be called after "parallel_run"

Reply:
Hi @wangxin 
I did fixes, it works. 
Thanks